scripts/data_files/vs2010-app-template.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
You're also removing the BOM (U+FEFF), here and in every file touched, which is not mentioned in the commit message. Is that intentional? Will VS accept those files without a BOM?

Reply:
Well-spotted! No, the change wasn't intentional. I don't know if the BOM is required, and since I don't know I don't want to change anything. I'll fix it in a rebase and pay closer attention to what my editor is doing.

Reply:
Looks like Emacs automatically removes a leading U+FEFF at the beginning of an XML file encoded in UTF-8. [It shouldn't be there](https://www.opentag.com/xfaq_enc.htm#enc_bom), but if Visual Studio prefers technically invalid XML, I'll keep the XML technically invalid.

Reply:
Thanks for fixing! `git show --patch -w` now shows an empty diff as it should